### PR TITLE
Final effect timing

### PR DESCRIPTION
**Pull request by alxhub**
Switches `effect`s to schedule within change detection instead of using the global microtask queue.

### CODE REVIEW

File: packages/core/src/application_ref.ts [hunk 1]
```diff
@@ -729,6 +730,7 @@ export class ApplicationRef {
   /** @internal */
   _views: InternalViewRef[] = [];
   private readonly internalErrorHandler = inject(INTERNAL_APPLICATION_ERROR_HANDLER);
+  private readonly effectManager = inject(EffectManager);
```
Comment (pkozlowski-opensource) [inline packages/core/src/application_ref.ts:733]: unused, removing

**Review by pkozlowski-opensource — APPROVED** [review]
LGTM

Reviewed-for: public-api 
Reviewed-for: fw-core

**Review by pkozlowski-opensource — APPROVED** [review]
LGTM

Reviewed-for: public-api 
Reviewed-for: fw-core 
Reviewed-for: fw-security

**Review by josephperrott — APPROVED** [review]
LGTM

Reviewed-for: fw-security

**Pull request closed without merging**